PdfScribeInstallCustomAction/Properties/Resources.resx

```diff
@@ -0,0 +1,228 @@
+  </data>
+  <data name="InfoInstallPrinterDriverFailed" xml:space="preserve">
+    <value>Printer driver installation failed.</value>
+  </data>
+  <data name="InfoInstallPrinterFailed" xml:space="preserve">
+    <value>Could not add printer.</value>
+  </data>
+  <data name="InstallRollBackFailureAtFunction" xml:space="preserve">
+    <value>Partial uninstallation failure. Function {0} returned false.</value>
+  </data>
+  <data name="NativeCouldNotEnable64Redirection" xml:space="preserve">
+    <value>Could not enable 64-bit file system redirection.</value>
+  </data>
+  <data name="NativeCouldNotRevert64Redirection" xml:space="preserve">
+    <value>Could not revert 64-bit file system redirection.</value>
+  </data>
+  <data name="PortMonitorAddFailed" xml:space="preserve">
+    <value>Could not add port monitor {0}</value>
+  </data>
+  <data name="PortMonitorAlreadyInstalled" xml:space="preserve">
+    <value>Port monitor {0} already installed.</value>
+  </data>
+  <data name="PortMonitorDllRemoveFailed" xml:space="preserve">
+    <value>Could not remove port monitor dll.</value>
+  </data>
+  <data name="PortMonitorRetrieveFailed" xml:space="preserve">
+    <value>Could not enumerate port monitors.</value>
+  </data>
+  <data name="PortMonitorSuccessInstalled" xml:space="preserve">
+    <value>Port monitor successfully installed.</value>
+  </data>
+  <data name="PrinterConfigured" xml:space="preserve">
+    <value>Printer configured.</value>
+  </data>
+  <data name="PrinterDriverAddFailed" xml:space="preserve">
+    <value>Could not add Aivika printer driver.</value>
+  </data>
+  <data name="PrinterDriverExisted" xml:space="preserve">
+    <value>Printer drivers copied or already exist.</value>
+  </data>
+  <data name="PrinterDriverInstalled" xml:space="preserve">
+    <value>Printer driver installed.</value>
+  </data>
+  <data name="PrinterDriverRemoveFailed" xml:space="preserve">
+    <value>Could not remove Aivika printer driver.</value>
+  </data>
+  <data name="PrinterDriverRetrieveFailed" xml:space="preserve">
+    <value>Could not retrieve printer driver directory.</value>
+  </data>
+  <data name="PrinterDriversEnumerateFailed" xml:space="preserve">
+    <value>Could not enumerate printer drivers.</value>
+  </data>
+  <data name="RedirectionPortAdded" xml:space="preserve">
+    <value>Redirection port added.</value>
+  </data>
+  <data name="RegistryConfigNotAdded" xml:space="preserve">
+    <value>Could not add port configuration to registry. Exception message: {0}</value>
+  </data>
+  <data name="RegistryConfigNotDeleted" xml:space="preserve">
+    <value>Could not delete port configuration from registry. Exception message: {0}</value>
+  </data>
+  <data name="VirtualPrinterAddFailed" xml:space="preserve">
+    <value>Could not add Aivika virtual printer.</value>
+  </data>
+  <data name="VirtualPrinterDeleteFailed" xml:space="preserve">
+    <value>Could not delete Aivika virtual printer.</value>
```

Comment:
Casing - Aivika Virtual Printer. Please adjust other references in this file too.

Reply:
noted.